src/ru.json

```
@@ -494,28 +509,28 @@
         "memoryCheckButton": "Сохранено. Проверить память"
     },
     "multiWallets": {
+        "addWallet": "Добавить кошелек",
         "create": "Создать кошелек",
         "defaultWalletName": "Мой Surf",
         "editWallet": "Редактировать кошелек",
         "manage": "Управлять",
         "manageWallets": "Управлять кошельками",
         "migrate": "Перенести кошелек в Surf",
         "name": "Имя",
+        "nameTaken": "Выберите другое имя. Это уже занято",
         "newWalletName": "Кошелек",
         "pair": "Привязать аппаратный кошелек к учетной записи",
         "removeWallet": "Удалить кошелек",
         "restore": "Восстановить кошелек",
-        "addWallet": "Добавить кошелек",
-        "restoreHint": "Surf currently supports only verified native wallets. You will not be able to open any other type of wallet.",
+        "restoreHint": "Вы можете восстановить кошелек, созданный только в Surf",
```

Переведено не полностью: 
```suggestion
        "restoreHint": "В настоящее время поддерживается только восстановление кошельков, созданных в Surf.",
```